sparql/sparql12/index.html

```diff
@@ -62,7 +62,7 @@ <h1 property='rdfs:label'>SPARQL 1.2 tests</h1>
     <div>
       <h2 id='abstract'>Abstract</h2>
       <p property='rdfs:comment'>
-        <p>These test suites are a product of the <a href="">W3C RDF-star Working Group</a> as well as the RDF-star Interest Group within the W3C RDF-DEV Community Group, and has been maintained by the <a href="https://www.w3.org/community/rdf-tests/">RDF Test Curation Community Group</a> at <a href="https://github.com/w3c/rdf-tests/tree/main/sparql/sparql12/">https://github.com/w3c/rdf-tests/tree/main/sparql/sparql12</a>. Conformance with SPARQL 1.2 specifications can be determined via successfully running the tests for relevant specifications along with the relevant SPARQL 1.1 tests.</p>
+        <p>These test suites are a product of the <a href="">W3C RDF 1.2 Working Group</a> as well as the RDF 1.2 Interest Group within the W3C RDF-DEV Community Group, and has been maintained by the <a href="https://www.w3.org/community/rdf-tests/">RDF Test Curation Community Group</a> at <a href="https://github.com/w3c/rdf-tests/tree/main/sparql/sparql12/">https://github.com/w3c/rdf-tests/tree/main/sparql/sparql12</a>. Conformance with SPARQL 1.2 specifications can be determined via successfully running the tests for relevant specifications along with the relevant SPARQL 1.1 tests.</p>
```

Comment:
The correct name is "RDF & SPARQL Working Group"

see https://www.w3.org/groups/wg/rdf-star/